### ISSUE TITLE
when will the full model release?

### ISSUE THREAD

**Issue by jinfagang** [aim-uofa/Poseur#5]
hi, it's being 3 month, wehen will the full model release?

**Comment by maowayne123** [comment]
Hi, all the models weight and cig file have been released, pls check read me. However, due to the recent update of mmpose, some model may have worse performance when you retraining the models. We now focusing on it this issue.

**Comment by jinfagang** [comment]
@maowayne123 thank u. But your repo is based on certain commit isn't it? does this repo's mmpose able to reproduce your papre result？

**Comment by maowayne123** [comment]
yes, our ori repo is based on mmpose. But i think the current mmpose modify some code. that causes some diffence, so the result of the current repo may differrent from the ori one. we will fix it soon. but the released model weight (in readme) can produce the same result with our paper. that means the inference code is perfect.

**Comment by maowayne123** [comment]
For the training code, we only checked the mbn v2 model, the model acc is lower than our paper. but we didn't checked other model, including resnet-50. maybe you can check it.
